aggregate_file.py

```diff
@@ -0,0 +1,133 @@
+import ast
+import os
+import re
+from collections import namedtuple
+
+
+def get_import_tuples(path):
```

Comment:
Let's add some docstring here, especially what it's returning. Having some example would help people understand more since `ast` is not a widely used library

Reply:
And type hints please as always

Reply:
Sure!